### DISCUSSION TITLE
PyroSim 2024 exporting fds not to source folder ( minor inconvenience )

### DISCUSSION THREAD

**Discussion by junyiiing** [firemodels/fds#13713] [Q&A]
I used PyroSim 2021 really frequently and every time when I'm exporting to fds , it automatically directs me to export into the source folder ( the same folder where I saved my .psm file)
Ever since I switched to PyroSim 2024, it does not direct me to the source folder but to my last exported fds location or to desktop instead. 
I know its not a HUGE issue, just a minor inconvenience especially when working in a folder that is deep inside the drive. 
Anyone here has a solution to this small problem?

**Comment by Vojta-Salek** [comment]
I would try contacting [Thunderhead](https://www.thunderheadeng.com/pyrosim/), this doesn't sound like FDS related problem.
